packages/button/README.md

```diff
@@ -388,7 +390,13 @@ mounts. This is useful for setting focus to a specific sp-button when a
 popover or dialog opens.
 
 ```html
-<sp-button autofocus>Confirm</sp-button>
+<sp-button id="trigger">Open</sp-button>
```

Comment:
Without putting this in a dialog, every time we open this docs page, it autofocuses to this section.